pkg/apis/sources/v1alpha1/azureeventgrid_types.go

```diff
@@ -95,7 +106,7 @@ type AzureEventGridSourceStatus struct {
 
 	// Resource ID of the Event Hubs instance that is currently receiving
 	// events from the Azure Event Grid subscription.
-	EventHubID *EventHubResourceID `json:"eventHubID,omitempty"`
```

Comment:
May want to have a `+optional` annotation here as well.

Reply:
No because this field was _removed_ 🙂 
And if I'm not mistaken the "+optional" convention applies only to Spec fields, because Status fields are read-only from a user's perspective.

Reply:
Understood on the status portion. The placement of this comment was with respect to the addition of `AzureResourceID` and not the removal of `EventHubResourceID`.

Reply:
I realized that only after and edited my comment, sorry for the misunderstanding!